_dashboards/discover/visualizing-ad.md

```diff
@@ -0,0 +1,158 @@
+1. Select the ellipsis icon from the **<name>** visualization pane, and then from the **Options** pop-up window, choose **Anomaly Detection** > **Add anomaly detector**.
+1.  Select **Create new detector**.
+1. From the Add anomaly detector window, define the metrics and set thresholds under **Detector details** and **Model features**. To view the visualization within this window, toggle the **Show visualization** button.
+1. _Optional_: Under **Detector details**, select <option> from the dropdown menu to configure the detector details.
+1. _Optional_: Under **Model features**, select <option> from the dropdown menu to configure the evaluation metrics. You can add and delete model features as best suited your use case and data, but you are limited to five model features for that detector.
+1. Select **Create detector**.
+
+Once you have created a new detector, the detector is added to the visualization, as shown in the following image.  
+
+<insert UI>
+
+### Creating an alerting monitor from the detector
+
+You can set up an alerting monitor based on your detector in one flow from the visualization. To set up an alerting monitor based on the detector created in the preceding steps, follow these steps:
+
+1. In the notification pop-up window that appears after you have created the detector, select **Set up alerts**.
+2. Follow the steps described in [Creating alerting monitors](#creating-alerting-monitors).
+
+You now have an alerting monitor and anomaly detector associated with the visualization.
+
+<insert UI>
+
+## Adding associated anomaly detectors
+
+You can add existing detectors to a visualization using the Discover application instead of Stack Management, giving you a single user interface where can add, view, and edit monitors and monitor details. Continuing with the visualization and dashboard in the preceding tutorial, follow these steps to associate an anomaly detector to a visualization: 
+ 
+1. From the dashboard, select the ellipsis icon on the **<name>**, then **Anomaly Detection**.
+2. Select **Associate a detector**.
+3. From the **Select detector to associate** dropdown menu, view the list of existing monitors and then select the desired monitor. In this example, select **<name>**. Note that basic information about the monitor is summarized in the window. To view more comprehensive details, select **View detector page**, which opens the detector details from the Anomaly Detection plugin page. 
+4. Verify you have selected the appropriate detector, and then select **Associate detector**. 
+
+An existing detector is now associated to the visualization, as shown in the following image:
+
+<insert UI>
+
+## Refreshing the visualization
+
+Depending on the threshold settings, the visualization refreshes at the set interval. To manually refresh the visualization to display real-time alerts and anomalies, select **Refresh** on the dashboard window.
+
+## Considerations and limitations
```

Comment:
We can also mention that by default, only 10 plugin resources per visualization is supported. This can be configured and changed in the advanced settings.

I mentioned it in the above comments a bit too, but we can provide some more details on
- the visualization eligibility requirements to support the feature
- the eligibility of alerting monitors that can be associated
- the eligibility of anomaly detectors that can be associated.

For anomaly detectors, the limitation is that detectors with category fields (we call them high-cardinality detectors') are not eligible.

cc @lezzago to help provide details on the others.

Reply:
For the Alerting plugin, we only allow creating simple Query Level monitors. Also we can reiterate the visualization eligibility requirements.